backend/dataall/core/organizations/db/organization_repositories.py

```diff
@@ -106,6 +106,19 @@ def query_organization_groups(session, uri, filter) -> Query:
             )
         return query.order_by(models.OrganizationGroup.groupUri)
 
+    @staticmethod
+    def is_group_invited(session, uri, group) -> bool:
+        return (
+            session.query(models.OrganizationGroup)
+            .filter(
+                and_(
+                    models.OrganizationGroup.organizationUri == uri,
+                    models.OrganizationGroup.groupUri == group,
+                )
+            )
+            .count()
+        ) > 0
```

Comment:
nit: Don't need to specify `>0` - if returns a value greather than 0 will evaluate as True in the if condition either way